src/core_plugins/vega/public/vega_view/vega_base_view.js

```diff
@@ -8,6 +8,13 @@ vega.scheme('elastic',
   ['#00B3A4', '#3185FC', '#DB1374', '#490092', '#FEB6DB', '#F98510', '#E6C220', '#BFA180', '#920000', '#461A0A']
 );
 
+export const bypassToken = Symbol();
```

Comment:
for a little bit more safety, could we NOT export the actual token here? We could use the `bypassExternalUrlCheck` function in the test, where it's currently imported to create that inner object.